_pages/about-us/teams/products-and-platforms.md

```diff
@@ -51,10 +53,10 @@ If you're a P&P project lead, expect to:
 
 * Report weekly on your project's status, blockers, and financial status via the weekly meeting agenda.
 
-* Be involved in the staffing process
+* Be involved in the staffing process.
```

Comment:
The most accurate and up to date description of the staffing process of which I am aware is https://github.com/18F/staffing-and-resources/wiki.  Does that fit the need?